config.js

```diff
@@ -389,11 +386,11 @@ if (argv.botid || argv.bot || argv.id || argv.minrankedhandicap || argv.maxranke
 
     if (argv.banunranked) {
         for (let i of argv.banunranked.split(',')) {
-            banned_unranked_users[i] = true;
+            exports.banned_unranked_users[i] = true;
         }
     }
 
```

Comment:
Would it make sense to do something like 
```
exports.banned_unranked_users = exports.banned_unranked_users.concat(banned_users)
```
?

That way you would no longer need to check both lists later on.

Reply:
interesting too, but first priority is to fix not working things or add new features :+1:

Reply:
This has been fixed in devel or maybe from Big to Small PR 1, I don't know.